dracut-module-setup.sh

```diff
@@ -1041,6 +1041,9 @@ install() {
     # Also redirect dracut-emergency to kdump error handler
     ln_r "$systemdsystemunitdir/emergency.service" "$systemdsystemunitdir/dracut-emergency.service"
 
+    # Disable ostree as we only need the physical root
+    systemctl -q --root "$initdir" mask ostree-prepare-root.service
```

Comment:
If you do this, this means that the crash dump file will be written in the "real root" partition (`/sysroot`) and not in the actual location set in the configuration file.

Reply:
I did test this. I think the reason it works is because the earlier code is resolving through the bind mount to the real sysroot target.